els_code_actions.erl

```diff
@@ -0,0 +1,120 @@
+-module(els_code_actions).
```

Comment:
is this module just accidentally part of this commit?

Reply:
Oops good catch. Belongs to another pr